src/test/java/com/google/gcp_variant_transforms/entity/VariantCallTest.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.entity;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class VariantCallTest {
+  private static final List<Integer> GENOTYPE = Arrays.asList(0, 71, 84);
+  private static final String SAMPLE_NAME = "NA00001";
+  private static final String PHASE_SET = "*";
+
+  private VariantCall testVariantCall;
+  private Map<String, Object> info = new HashMap<>();
+
+  @Before
+  public void buildTestInfoMap() {
+    info.put("GQ", 21);
+    info.put("DP", 6);
+    info.put("HQ", Arrays.asList(23, 27));
+  }
+
+  @Before
+  public void buildTestVariantCall() {
+    testVariantCall = new VariantCall(SAMPLE_NAME, GENOTYPE, PHASE_SET, info);
+  }
+
+  @Test
+  public void testGetSampleName_whenComparingString_thenTrue() {
+    assertThat(testVariantCall.getSampleName().equals(SAMPLE_NAME)).isTrue();
```

Comment:
We shouldn't have checks like this - isTrue is to be used only for boolean verification. It should be something like

`assertThat(testVariantCall).getSampleName().contains(SAMPLE_NAME);`

or something similar. Here and in the rest of the tests.

Reply:
I think equals() is a boolean verification to check if the string is same as what we expected. I am not sure if I have got what you mean since contains() will not verify string if the string contains NAME but also have some prefix or suffix